tabs/configuration.js

```diff
@@ -410,28 +428,61 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
         // fill magnetometer
         $('input[name="mag_declination"]').val(MISC.mag_declination.toFixed(2));
 
-        //fill motor disarm params and FC loop time        
+        //fill motor disarm params and FC loop time
         if(semver.gte(CONFIG.apiVersion, "1.8.0")) {
             $('input[name="autodisarmdelay"]').val(ARMING_CONFIG.auto_disarm_delay);
             $('input[id="disarmkillswitch"]').prop('checked', ARMING_CONFIG.disarm_kill_switch !== 0);
-            $('div.disarm').show();            
+            $('div.disarm').show();
 
             $('div.cycles').show();
         }
-        
+
         // fill throttle
         $('input[name="midrc"]').val(MISC.midrc);
         $('input[name="minthrottle"]').val(MISC.minthrottle);
         $('input[name="maxthrottle"]').val(MISC.maxthrottle);
         $('input[name="mincommand"]').val(MISC.mincommand);
-        
+
         // fill battery
+        var batteryMeterTypes = [
+            'None', //Do not show in GUI
```

Comment:
`None` should be removed altogether, see https://github.com/betaflight/betaflight/pull/1646.